src/scripts/content/intercom.js

```diff
@@ -31,3 +31,21 @@ togglbutton.render('.conversation__card__content-expanded__controls .inbox__conv
 
     elem.appendChild(link);
   });
+
+togglbutton.render('.articles__editor__header-text:not(.toggl)',
+  { observe: true },
+  function (elem) {
+    const descriptionSelector = () => {
+      const description = elem.textContent;
+      return description ? description.trim() : '';
+    };
+
+    const link = togglbutton.createTimerLink({
+      className: 'intercom',
+      description: descriptionSelector,
+      buttonType: 'minimal'
+    });
+    link.style.margin = '3px 15px';
```

Comment:
Can't we include it in css instead? 🙏

Reply:
I would refrain from adding more new styles to that common CSS file,
we should ideally allow CSS files for the content scripts and have them live next to the script itself,
having styles in content scripts seems like an acceptable alternative until we move towards independent CSS files.

The monolith CSS file is difficult to reason about and is easy to miss, when refactoring/adding/removing integrations

Reply:
@rylek90 same as :point_up:, I didn't want to add another single use class in there. Are you ok with keeping it in the script?

Reply:
> we should ideally allow CSS files for the content scripts and have them live next to the script itself

This sounds great 🙌 

> Are you ok with keeping it in the script?

Yeah, sure 😅